Makefile

```diff
@@ -14,8 +14,8 @@
 GO := go
 DOCKER := docker
 TAG?=$(shell git rev-parse HEAD)
-REGISTRY?=gcr.io/google-containers
-IMAGE=node-termination-handler
+REGISTRY?=gcr.io/k8s-image-staging
+IMAGE=gke-node-termination-handler
```

Comment:
Why are we adding gke-? Would this not work on GCE? Should we use gcp-?

Reply:
Ping on this. LGTM otherwise.

Reply:
The images that are specific to gke stored in `k8s-image-staging` project seem to all have a `gke-` prefix. This is to differentiate between GCP specific and pure OSS k8s components.